### PR TITLE
Multiple ecoverse support, identifiers refresh

### PR DESCRIPTION
**Pull request by techsmyth**

### CODE REVIEW

File: src/populators/ecoverse-populator.ts [hunk 1]
```diff
@@ -31,98 +32,124 @@ export class EcoversePopulator extends AbstractPopulator {
...
+      ID: ecoverseData.nameID,
+      displayName: ecoverseData.displayName,
+      hostID: ecoverseData.host,
+      context: {
+        background: ecoverseData.background,
+        impact: ecoverseData.impact,
+        tagline: ecoverseData.tagline,
+        vision: ecoverseData.vision,
+        who: ecoverseData.who,
+        // references: [
```
Comment (valentinyanakiev) [inline src/populators/ecoverse-populator.ts:93]: Why are the references commented out?

File: src/populators/ecoverse-populator.ts [hunk 2]
```diff
@@ -31,98 +32,124 @@ export class EcoversePopulator extends AbstractPopulator {
...
+        //   {
+        //     name: 'website',
```
Comment (valentinyanakiev) [inline src/populators/ecoverse-populator.ts:95]: Why are the references commented out (as above)?

File: src/populators/ecoverse-populator.ts [hunk 3]
```diff
@@ -31,98 +32,124 @@ export class EcoversePopulator extends AbstractPopulator {
...
+      nameID: ecoverseData.nameID,
+      displayName: ecoverseData.displayName,
+      hostID: ecoverseData.host,
+      context: {
+        background: ecoverseData.background,
+        impact: ecoverseData.impact,
+        tagline: ecoverseData.tagline,
+        vision: ecoverseData.vision,
+        who: ecoverseData.who,
+        // references: [
```
Comment (valentinyanakiev) [inline src/populators/ecoverse-populator.ts:127]: Same as above.

**Review by valentinyanakiev — COMMENTED** [review]
If comments are intentionally left out we can merge this through but in general, it's better not to leave commented code out.

File: src/populators/ecoverse-populator.ts (hunk 3 above)
Comment (techsmyth) [reply src/populators/ecoverse-populator.ts:127]: These code pieces were already commented out, and did not have bandwidth to look at the rational so just left them there. Suggest to leave for now as higher priorities...

**Review by valentinyanakiev — APPROVED** [review]

**Pull request merged**